gnmi_server/server_test.go

```diff
@@ -3095,6 +3110,10 @@ func TestConnectionsKeepAlive(t *testing.T) {
 }
 
 func TestClient(t *testing.T) {
+    var mu_deinit_done sync.Mutex
+    var mu_hb_done sync.Mutex
+	var mu_idx_done sync.Mutex
```

Comment:
Please rename these variables. Also, the indentation looks off.

https://google.github.io/styleguide/go/decisions#indent-error-flow:~:text=areas%20within%20naming.-,Underscores,-Names%20in%20Go

Reply:
Thanks for your comments, I've fixed all indents and naming issues.